concepts/timeline.md

```diff
@@ -29,6 +38,16 @@ They should be encouraged to be lead in at least 1 project, but this is not requ
 
 Money should start coming in this time, if appreciation of crypto assets didn't extend that period
 
+```{note}
+Can maybe clarify that part, do you expect the project to be mature and revenue-generating after 2 yrs of definition and then 2 yrs of production? What level of revenue generation to expect from the project itself? 
+
+If a project is production-grade and can add value, business dev/marketing/sales can theoretically scale revenue. What is a minimum set of tasks necessary to 'test a market', at least in a very basic form?, and know if more work/money needs to be invested for comercialization / donation-based / consulting based.
+
+I understand you want to avoid going into a full business or financial plan, but developing on the financials can give a more solid basis to the project.
```

Comment:
this is as of now voluntarily under-specified as a potential business plan should be decided and voted upon by the members of the collective, not something that should be pre-existing to the collective.

The idea is to have an initial amount of seed money that would be enough to pay the salaries of 5 persons during 5 years (approximately), and at that time the project should be self-sustaining or discontinued (or scaled down to keep the organization but without the salaries) if self-sustainability cannot be achieved.